tools/upload_to_anaconda_staging.sh

```diff
@@ -16,7 +16,7 @@ else
     ls -lh builds/openblas*.zip
 
     anaconda -t $OPENBLAS_LIBS_STAGING_UPLOAD_TOKEN upload \
```

Comment:
this needs to change to the new name `ANACONDA_SCIENTIFIC_PYTHON_UPLOAD`, both here and in the travis run